### PR TITLE
metric/resource_control: append write dimension of the RU max metric

### PR DESCRIPTION
**Pull request by HuSharp**
### What problem does this PR solve?
append write dimension of the RU max metric

before 
image

after

image

Issue Number: Ref #54434

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [x] I checked and no code files have been changed.
  > 

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by nolouch — APPROVED** [review]

File: pkg/metrics/grafana/tidb_resource_control.json [hunk 1]
```diff
@@ -223,7 +223,7 @@
           "targets": [
             {
               "editorMode": "code",
-              "expr": "sum(resource_manager_resource_unit_read_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
+              "expr": "sum(resource_manager_resource_unit_read_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group) + sum(resource_manager_resource_unit_write_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
```
Comment (nolouch) [inline pkg/metrics/grafana/tidb_resource_control.json:226]: We can display the type separately. The maximum is the highest RU in the last 20 seconds. These may not occur simultaneously, so it's better to show them separately.

Comment (HuSharp) [reply pkg/metrics/grafana/tidb_resource_control.json]: fixed

**Comment by HuSharp** [comment]
@JmPotato PTAL, thx!

File: pkg/metrics/grafana/tidb_resource_control.json [hunk 2]
```diff
@@ -225,7 +225,15 @@
               "editorMode": "code",
               "expr": "sum(resource_manager_resource_unit_read_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
               "hide": false,
-              "legendFormat": "{{resource_group}}",
+              "legendFormat": "{{resource_group}}-read",
+              "range": true,
+              "refId": "C"
```
Comment (JmPotato) [inline pkg/metrics/grafana/tidb_resource_control.json:230]: A different `refID` should be used here.

File: pkg/metrics/grafana/tidb_resource_control.json [hunk 3]
```diff
@@ -225,7 +225,15 @@
...
+            },
+            {
+              "editorMode": "code",
+              "expr": "sum(resource_manager_resource_unit_write_request_unit_max_per_sec{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\"}) by (resource_group)",
+              "hide": false,
+              "legendFormat": "{{resource_group}}-write",
               "range": true,
               "refId": "C"
```
Comment (JmPotato) [inline pkg/metrics/grafana/tidb_resource_control.json:238]: Ditto.

File: pkg/metrics/grafana/tidb_resource_control.json (hunk 2 above)
Comment (HuSharp) [reply pkg/metrics/grafana/tidb_resource_control.json]: There seem to be only two in the this panel, I changed it to A and B.

**Comment by nolouch** [comment]
PTAL @glorv

**Review by JmPotato — APPROVED** [review]

**Review by tangenta — APPROVED** [review]

**Pull request merged**